src/Shared/Implementation/ITelemetryModuleConfigurator.cs

```diff
@@ -1,6 +1,14 @@
-﻿namespace Microsoft.ApplicationInsights.AspNetCore
+﻿#if AI_ASPNETCORE_WEB
+    namespace Microsoft.ApplicationInsights.AspNetCore
+#else
+    namespace Microsoft.ApplicationInsights.WorkerService
+#endif
 {
+#if AI_ASPNETCORE_WEB
     using Microsoft.ApplicationInsights.AspNetCore.Extensions;
+#else
+    using Microsoft.ApplicationInsights.WorkerService;
+#endif
```

Comment:
I'm seeing a lot of preprocessors around the usings. 
Would it be easier to have a new namespace for these Shared files?
Ex: "Microsoft.ApplicationInsights.NetCoreShared"